lib/scrabble-project/player.rb

```diff
@@ -0,0 +1,32 @@
+module Scrabble
+
+  class Player
+
+    attr_accessor :name, :plays, :total_score
+
+    def initialize(name)
+      @name = name
+      @plays = []
+      @won = false
+      @total_score = 0
+    end
+
+    def play(word)
+      @plays.push(word)
+      return @plays
+    end
+
+    def total_score     # This method doesn't work! I got as far as making it, but not as far as making it work. 
+      @plays.each do |word|
+      @total_score += (Scrabble::Scrabble.score(word))
+      end
+      return @total_score
+    end
```

Comment:
You're on the right track! Think about it this way: Since `@total_score` is an instance variable, its state gets stored on the instance of Player. In other words, if you call the method `total_score` on a Player instance, it will sum up all the points currently in `@plays` and store it in `@total_score`. But what happens if the player has a few more plays, and then you call `total_score` again? 

Hint: Do you need `@total_score` as an instance variable at all?